pkg/didcomm/common/service/event.go

```diff
@@ -56,3 +75,12 @@ type Event interface {
 	// UnregisterMsgEvent on protocol messages. Refer RegisterMsgEvent().
 	UnregisterMsgEvent(ch chan<- StateMsg) error
 }
+
+// DIDExchangeEvent properties related api.
+type DIDExchangeEvent interface {
```

Comment:
Suggest move this to the didexchange package.

Reply:
[Moved](https://github.com/hyperledger/aries-framework-go/compare/ebaa2c60d8b2fce620d906c585bfb49c74ba8111..cc46c9e091f6091fb11e5c1467352ad67d28c0ea#diff-79de382a052bbdb4213b9dd373b0e2f6R10).